src/controller/java/generated/java/chip/devicecontroller/cluster/eventstructs/AccessControlClusterAccessControlEntryChangedEvent.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package chip.devicecontroller.cluster.eventstructs
+
+import chip.devicecontroller.cluster.*
+import chip.tlv.AnonymousTag
+import chip.tlv.ContextSpecificTag
+import chip.tlv.Tag
+import chip.tlv.TlvParsingException
+import chip.tlv.TlvReader
+import chip.tlv.TlvWriter
+
+import java.util.Optional
+
+class AccessControlClusterAccessControlEntryChangedEvent (
+    val adminNodeID: Long?,
+    val adminPasscodeID: Int?,
+    val changeType: Int,
+    val latestValue: chip.devicecontroller.cluster.structs.AccessControlClusterAccessControlEntryStruct?,
+    val fabricIndex: Int) {
+  override fun toString() : String {
+    val builder: StringBuilder = StringBuilder()
```

Comment:
using https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/build-string.html is probably more kotlinLike:

```kotlin
override fun toString() : String = buildString {
   append("AccessControlClusterAccessControlEntryChangedEvent {\n")
   append("\tadminNodeID : $adminNodeID\n")
   // ...
}
```

Reply:
I understand. I'll fix it.